docs/dev/maintenance/BackportTranslations.md

```diff
@@ -0,0 +1,28 @@
+# Backporting translations
+
+It is usual that at the time a new stable release is created (and thus there is a new, separate code-branch for it that remains feature-frozen) the
+translations into the various languages is incomplete and after the separation from the master branch has been performed one or more translations
+receive updates.
```

Comment:
I have a very hard time understanding this sentence.

Reply:
I reformulated it. Is it better now?